CHANGELOG.rst

```diff
@@ -1,6 +1,50 @@
 Changelog
 =========
 
+0.7.0
+
+MLflow 0.7.0 introduces several major features:
+
+- An R client API, available on CRAN
+- Support for deleting runs
+- UI support for adding notes to a run
+
+The release also includes bugfixes and improvements across the Python and Java clients, tracking UI,
+and documentation.
+
+Breaking changes:
+- [Python] The per-flavor implementation of load_pyfunc has been made private (#539, @tomasatdatabricks)
```

Comment:
Btw, the float --> double change went in (#566), so we may want to mention that as a breaking Java API change.